cl/scrapers/management/commands/clone_from_cl.py

```diff
@@ -0,0 +1,406 @@
+        except model.DoesNotExist:
+            pass
+
+        cluster_url = f"{cluster_endpoint}{cluster_id}/"
+        cluster_datum = session.get(cluster_url, timeout=60).json()
+        docket_id = cluster_datum["docket"].split("/")[7]
+        docket = clone_docket(session, [docket_id])[0]
+        citation_data = cluster_datum["citations"]
+        sub_opinions_data = cluster_datum["sub_opinions"]
+        # delete unneeded fields
+        for f in [
+            "resource_uri",
+            "docket",
+            "citations",
+            "sub_opinions",
+            "absolute_url",
+            "panel",
+            "non_participating_judges",
+        ]:
+            del cluster_datum[f]
+
+        # Assign docket pk in cluster data
+        cluster_datum["docket_id"] = docket.pk
+
+        prepared_opinion_data = []
+        added_opinions_ids = []
+
+        for op in sub_opinions_data:
+            # Get opinion from api
+            op_data = session.get(op, timeout=60).json()
+            # Delete fields with fk or m2m relations or unneeded fields
+            for f in [
+                "opinions_cited",
+                "cluster",
+                "absolute_url",
+                "resource_uri",
+                "author",
+                "joined_by",
+            ]:
+                del op_data[f]
+            # Append new data
+            prepared_opinion_data.append(op_data)
+
+        with transaction.atomic():
+            # Create opinion cluster
+            opinion_cluster = model.objects.create(**cluster_datum)
+
+            for cite_data in citation_data:
+                # Create citations
+                cite_data["cluster_id"] = opinion_cluster.pk
+                Citation.objects.create(**cite_data)
+
+            for opinion_data in prepared_opinion_data:
+                # Update cluster_id in opinion's json
+                opinion_data["cluster_id"] = opinion_cluster.pk
+
+                # Create opinion
+                op = Opinion.objects.create(**opinion_data)
+
+                # Store created opinion id
+                added_opinions_ids.append(op.id)
+
+            opinion_clusters.append(opinion_cluster)
+            print(
+                ">> View cloned case here:",
+                reverse("view_case", args=[opinion_cluster.pk, docket.slug]),
+            )
+
+        # Add opinions to search engine
+        add_items_to_solr.delay(added_opinions_ids, "search.Opinion")
+
+    return opinion_clusters
+
+
+def clone_docket(
+    session: Session, docket_ids: list, object_type="search.Docket"
+):
+    """
+    Download docket data from courtlistener.com and add it to local
+    environment
+    :param session: a Requests session
+    :param docket_ids: a list of docket ids
+    :param object_type: Docket app name with model name
+    :return: list of docket objects
+    """
+
+    dockets = []
+
+    for docket_id in docket_ids:
+        print(f">> Cloning docket id: {docket_id}")
```

Comment:
I used to do a lot of these kinds of things with `>` in my log lines or fancy indentation, but it makes logs harder to parse automatically. I'd suggest simplifying to just:

```suggestion
        print(f"Cloning docket id: {docket_id}")
```